crates/cxx-qt-lib/src/types/qset/mod.rs

```diff
@@ -220,7 +220,7 @@ macro_rules! impl_qset_element {
             }
 
             unsafe fn get_unchecked(set: &QSet<Self>, pos: usize) -> &Self {
-                $module::get_unchecked(set, pos)
+                $module::get_unchecked(set, pos as isize)
```

Comment:
where should we perform the usize -> isize conversion? As Qt uses signed types.

Should we expose len() to rust as isize?

But the Iter trait still needs usize for other parts?

Reply:
Right Qt is weird with using a signed type here :-)

I think changing the get_unchecked to isize makes sense. The only thing i wondered was if the public `QSet::len(&self)` should return a `usize` ? As it might seem odd to a Rust developer that the size is signed.

However when we move onto QVector/QList etc they have index_of/append/remove/at/len which are all potentially `isize`. So do we want to be consistent in using `isize` like Qt or convert some to `usize`? :thinking:

Reply:
Right, i've gone with using `isize` everywhere except in the iterator internals.